src/webui/webapplication.cpp

```diff
@@ -774,6 +776,26 @@ void WebApplication::action_command_bottomPrio()
     BitTorrent::Session::instance()->bottomTorrentsPriority(hashes);
 }
 
+void WebApplication::action_command_rename()
+{
+    CHECK_URI(0);
+    CHECK_PARAMETERS("hash" << "name");
+
+    QString hash = request().posts["hash"];
+    QString name = request().posts["name"].trimmed();
+
+    BitTorrent::TorrentHandle *const torrent = BitTorrent::Session::instance()->findTorrent(hash);
+    if (torrent && !name.isEmpty()) {
+        name.replace(QRegularExpression("\r?\n|\r"), " ");
```

Comment:
[`setName()`](https://github.com/qbittorrent/qBittorrent/blob/master/src/base/bittorrent/torrenthandle.cpp#L1265) doesn't appear to do any such validation.